lib/model/trait/actee.js

```diff
@@ -1,3 +1,8 @@
+// The Actee Trait simply adds in a couple of methods convenient for dealing
```

Comment:
As you know, I strongly prefer avoiding words like "just" or "simply"!

Reply:
The goal here is to attempt to defuse the sentiment I keep seeing from you, which is fear about what this trait thing is and what it does, by emphasizing that has a very limited use and that _all it does_ is add a couple of methods. Nothing more. So the language attempts to minimize because it is, I think, appropriate to do so. If someone doesn't understand what a trait is, it probably doesn't actually matter. I'm not sure how to do this deëmphasis without resorting to words like "simply" or "just"..

Reply:
Ok.